packages/react-dom/src/__tests__/ReactDOMFizzSuppressHydrationWarning-test.js

```diff
@@ -248,9 +248,10 @@ describe('ReactDOMFizzServerHydrationWarning', () => {
         ]);
       }).toErrorDev(
         [
+          'Expected server HTML to contain a matching <span> in <span>',
```

Comment:
These were being covered up, making mismatches with prod behavior hard to debug and fix.